v1.2.0.html

```diff
@@ -1940,7 +1940,9 @@ <h3 class="no-num no-toc no-ref heading settled" id="profile-and-date"><span cla
    <div data-fill-with="spec-metadata">
     <dl>
      <dt>This version:
-     </dt><dd><a class="u-url" href="https://aomediacodec.github.io/av1-isobmff/">https://aomediacodec.github.io/av1-isobmff/</a>
+     <dd><a class="u-url" href="https://aomediacodec.github.io/av1-isobmff/v1.2.0.html">https://aomediacodec.github.io/av1-isobmff/v1.2.0.html</a>
```

Comment:
I looked at the other files (v1.0.0 and v1.1.0) and the dd/dt are not balanced. It might have been a bug of Bikeshed. Given that it renders fine, I'm not going to fix that.